### ISSUE TITLE
Floating window support for neovim

### ISSUE THREAD

**Issue by liquidz** [liquidz/vim-iced#96]
Just an idea
https://github.com/neovim/neovim/pull/6619

**Comment by liquidz** [comment]
```vim
if exists('*nvim_open_win')
  let s:bufid = bufnr('%')
  let s:opts = {
        \ 'relative': 'editor',
        \ 'row': 10,
        \ 'col': 20,
        \ }
  call nvim_open_win(s:bufid, v:true, 20, 10, s:opts)
endif
```
It seems useful to display evaluation result near cursor like (spac)emacs.

**Comment by liquidz** [comment]
Draft PR: #97
https://twitter.com/uochan/status/1101091700888367105

**Comment by liquidz** [comment]
Closed by #102
